commons/src/test/java/io/aiven/kafka/tieredstorage/commons/UniversalRemoteStorageManagerConfigTest.java

```diff
@@ -105,6 +107,17 @@ void objectStorageFactoryCorrectClass() {
             .hasMessage("Invalid value x for configuration object.storage.factory: Class x could not be found.");
     }
 
+    @Test
+    void validKeyPrefix() {
```

Comment:
Would be nice to have a test for the valid case as well

Reply:
Added